storages/python/python-tests/bson_test.py

```diff
@@ -0,0 +1,73 @@
+"""Script to test the DLite plugin 'bson.py' in Python."""
```

Comment:
Test files conventionally starts with "test_". Tools like pytest or nose will automatically find all python tests in files matching "test_*.py". It also makes sorting more logically